docs/_embed/zap_editor.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Zap Editor
+title: Pre-filled Zaps & Embedding the Zap Editor
```

Comment:
Looks like the file name is associated to the URL path, not the title - so this should be safe to update! This is what displays on the leftside table of context of our platform docs.